### PR TITLE
Fixed riot police in extendedCallWindow

### PR DESCRIPTION
**Pull request by 02Miki**
I'm not too familiar with regex, so it might be wrong, apologies if it is

### CODE REVIEW

File: src/modules/extendedCallWindow/i18n/it_IT.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ module.exports = {
             [/^Volanti?$/]: [7, 16, 15, 21],
             [/^Furgon[ie] artificieri$/]: [22],
             [/^Elicotter[io] della polizia?$/]: [13],
-            [/^furgon[ae] antisommossa$/]: [14],
+            [/^Unit(à)? antisommossa$/]: [14],
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/i18n/it_IT.js:112]: What are the possible values? Now `Unit` and `Unità` do match. Is it what you want?

Comment (02Miki) [reply src/modules/extendedCallWindow/i18n/it_IT.js:112]: the only one should be Unità

Comment (jxn-30) [reply src/modules/extendedCallWindow/i18n/it_IT.js:112]: Then you can use `/^Unità antisommossa$/`. You don't need any special characters.

Comment (02Miki) [reply src/modules/extendedCallWindow/i18n/it_IT.js:112]: would this apply to `/^Unit(à)? cinofila antidroga$/` as well? as it only needs to match Unità

Comment (jxn-30) [reply src/modules/extendedCallWindow/i18n/it_IT.js]: Yes. The `?` marks that the character (or group if encloses in parantheses `()`) is optional.
`Hello? World` matches `Hello World` and `Hell World`.
`He(llo)? World` matches `Hello World` and `He World`.

Comment (02Miki) [reply src/modules/extendedCallWindow/i18n/it_IT.js]: Oh, thanks, appreciate it

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**